ui/imports/shared/controls/delegates/LinkPreviewCardDelegate.qml

```diff
@@ -0,0 +1,72 @@
+import QtQuick 2.15
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+import shared.controls.chat 1.0
+
+LinkPreviewCard {
+    id: root
+
+    //Model properties
+    required property bool unfurled
+    required property bool empty
+    required property string url
+    required property bool immutable
+    required property int previewType
+    required property var standardPreview
+    required property var standardPreviewThumbnail
+    required property var statusContactPreview
+    required property var statusContactPreviewThumbnail
+    required property var statusCommunityPreview
+    required property var statusCommunityPreviewIcon
+    required property var statusCommunityPreviewBanner
+    required property var statusCommunityChannelPreview
+    required property var statusCommunityChannelCommunityPreview
+    required property var statusCommunityChannelCommunityPreviewIcon
+    required property var statusCommunityChannelCommunityPreviewBanner
```

Comment:
Cannot we strictly type those properties?

Reply:
Ah, ok, there are js objects, maybe we could leave some comments regarding expected properties for better readability of this component api.